packages/autotools-scripts/src/cli.ts

```diff
@@ -17,6 +18,18 @@ program
   ssrTest();
 });
 
+program
+.command('a11y')
+.description('test')
+.option('-p, --path <p>', 'project path')
```

Comment:
Do we need the project path? We can use `process.cwd()` that gives the working directory (which is the project directory). And all tools should probably support `--files` option which allows to control which subset of metafiles is loaded.